tests/Test/Compiler.hs

```diff
@@ -53,21 +52,21 @@ testIf handleResult filePaths =
   where
     setupTest filePath =
       do  source <- readFile filePath
-          let result = Compiler.compile "elm-lang" "core" source Map.empty
+          let result = Compiler.compile "elm-lang" "core" True source Map.empty
```

Comment:
If a file `isRoot` it is allowed to have ports. If it is a dependency, it cannot have ports.